service/src/main/java/bio/terra/externalcreds/controllers/OidcApiController.java

```diff
@@ -76,6 +80,7 @@ public ResponseEntity<LinkInfo> createLink(
             .userId(samUser.getSubjectId())
             .clientIP(request.getRemoteAddr());
 
+    Optional<LinkInfo> linkInfo;
     try {
       if (providerName.equals(Provider.RAS)) {
```

Comment:
Since `providerName` is an enum value, we can replace this `if/else` with a `switch` statement